Dockerfile

```diff
@@ -65,11 +68,14 @@ RUN chmod +x /usr/local/bin/optimize_workspace.sh
 RUN apk add --no-cache musl-dev
 
 ADD build_workspace build_workspace
+# What do we need this for?
+RUN echo "Installed targets:" && (rustup target list | grep installed) && \
+  export DEFAULT_TARGET="$(rustc -vV | grep 'host:' | cut -d' ' -f2)" && echo "Default target: $DEFAULT_TARGET"
```

Comment:
It's probably a remnant of when we were struggling with multi-arch support. Mostly for informative reasons. It looks safe to remove it.

Reply:
Okay, yeah, I agree. Let's remove them. It's all nice and clean now.